website/members/middleware.py

```diff
@@ -25,4 +25,6 @@ def __call__(self, request):
         # This also avoids unnecessary queries when request.member is not used
         request.member = SimpleLazyObject(lambda: get_member(request))
 
+        print("---------------\nHost:", request.get_host())
```

Comment:
Is this required? And regardless of that question: this could be using a python logger.